rs-matter/src/mdns/astro.rs

```diff
@@ -91,12 +90,16 @@ impl<'a> MdnsService<'a> {
     }
 
     /// Only for API-compatibility with builtin::MdnsRunner
-    pub async fn run_udp(&mut self, buffers: &mut MdnsUdpBuffers) -> Result<(), Error> {
+    pub async fn run_udp(&mut self, _buffers: &mut MdnsUdpBuffers) -> Result<(), Error> {
```

Comment:
Needs to be named just `run` now.

Reply:
To clarify, one of the metods - say - this one ^^^, should have exactly this signature:
```
    pub async fn run<D>(
        &self,
        _stack: &crate::transport::network::NetworkStack<D>,
        _buffers: &mut MdnsRunBuffers,
    ) -> Result<(), Error>
    where
        D: crate::transport::network::NetworkStackDriver,
    {
```